### PR TITLE
Implement help page - Closes #943

### LINKED ISSUE #943: Implement Help Page
### Expected behaviour
- There should be a page showing links to different lisk help resources lisk.chat, academy and help center
- https://taikonauten.atlassian.net/browse/L20-421

- Creating help page should include removing "Onboarding" from the settings page and then Go to "Settings" on your Sidebar to start the onboarding whenever you need. should be changed to `Go to "Help" on your Sidebar to start the onboarding whenever you need.

screen shot 2018-07-31 at 10 03 06

https://files.slack.com/files-pri/T3CBPJ17X-FC1LV7GUF/screen_shot_2018-08-01_at_10.12.42.png

### PR DESCRIPTION
**Pull request by faival**
### What was the problem?
- #943 
### Review checklist
- The PR solves #943 
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

**Comment by slaweet** [comment]
Creating help page should include removing "Onboarding" from the settings page and then `Go to "Settings" on your Sidebar to start the onboarding whenever you need.` should be changed to `Go to "Help" on your Sidebar to start the onboarding whenever you need.

File: src/components/help/help.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            <h2>
+              <FontIcon className={styles.headerIcon}>logo-icon</FontIcon>
+              {this.props.t('Help Center')}
+            </h2>
+            <div className={styles.articleContainer}>
+              <div>
+                <p>{this.props.t('Search for advice and answers in our extensive ')}
+                  {fAQIcon()}
+                  {this.props.t(' or get in touch in the ')}
+                  {chatIcon()}
```
Comment (slaweet) [inline src/components/help/help.js]: It was missing in the design, but there should be a dot `.` at the end of this sentance.

File: src/constants/help.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const links = {
+  helpCenter: 'https://help.lisk.io/',
+  explainBlockchain: 'https://lisk.io/academy/blockchain-basics/what-is-blockchain/eli5-blockchain',
+  isBlockchainSecure: 'https://lisk.io/academy/blockchain-basics/benefits-of-blockchain/blockchain-security',
+  seeMore: 'https://lisk.io/academy/blockchain-basics/how-does-blockchain-work',
+  liskChat: 'https://lisk.chat/home',
+  helpDeskChannel: 'https://lisk.chat/channel/help-desk',
+};
+export default links;
```
Comment (slaweet) [inline src/constants/help.js]: Great idea to extract the links here 👍

File: src/components/mainMenu/mainMenu.css [hunk 3]
```diff
@@ -182,6 +182,65 @@
...
+}
+
+.bottomTab {
+  background: var(--menu-background-color);
+  box-shadow: none;
+  width: 100% !important;
+  max-width: 100% !important;
+  border: none !important;
+  padding: 7px 10px !important;
+}
```
Comment (slaweet) [inline src/components/mainMenu/mainMenu.css]: Do we need all these `!important`s here? 
Can the selector rather be made more specific by moving `.bottomTab` inside `.bottomTabs` block in this file?

**Comment by faival** [comment]
pr changes applied. M bPoint taken care
